src/Comparator/BackwardsCompatibility/InterfaceBased/InterfaceBecameClass.php

```diff
@@ -31,4 +31,12 @@ public function compare(ReflectionClass $fromClass, ReflectionClass $toClass) :
         ),
         ]);
     }
+
+    /**
+     * According to the current state of the PHP ecosystem, we only have traits, interfaces and classes
+     */
+    private function isClass(ReflectionClass $class) : bool
```

Comment:
Does `ReflectionClass` really not have an `isClass` method... seems like an oversight >.<

Reply:
Doesn't have one, no. That's because we mimicked the existing `ext-reflection` API, without designing our own API from scratch upfront.